jobs/pool/query.graphql

```diff
@@ -298,7 +298,7 @@ query CurrentBlock($first: Int = 1000, $skip: Int = 0) {
 }
 
 query SteerVaults($first: Int = 1000, $skip: Int = 0) {
-  vaults(first: $first, skip: $skip) {
+  Steer_vaults(first: $first, skip: $skip) {
```

Comment:
what I feel is nicer is doing

`vaults: Steer_vaults(first...`

so that you can avoid doing that everywhere in the code later on